ethdb/leveldb/leveldb.go

```diff
@@ -149,12 +150,24 @@ func NewCustom(file string, namespace string, customize func(options *opt.Option
 	return ldb, nil
 }
 
+// NewCustom returns a wrapped LevelDB object with an in-memory storage.
```

Comment:
Do we want to move this additions to a new file, to prevent conflicts with upstream? We were doing that before, I'm not sure if we want to continue doing that or not.

Reply:
s/NewCustom/NewInMemory

Reply:
Good eye, thanks! Yes, moved to a new file, thanks for pointing that out.